acceptance/gossip_peerings_test.go

```diff
@@ -141,6 +141,8 @@ func TestGossipPeerings(t *testing.T) {
 // re-bootstrapped after a time when all nodes were down
 // simultaneously.
 func TestGossipRestart(t *testing.T) {
+	t.Skipf("TODO(pmattis): #4445")
```

Comment:
I think it's the same root cause as the flakiness for the status server test as well (#4512).